scripts/limonade/src/index.ts

```diff
@@ -197,6 +267,71 @@ async function run() {
     }, argv.solanaWebsocketTimeout);
   });
 
+  if (priceConfigs.length > 0) {
+    const hermesClient = new HermesClient(argv.hermesEndpoint, {});
+
+    const eventSource = await hermesClient.getPriceUpdatesStream(
+      priceConfigs.map((priceConfig) => priceConfig.pythFeedId),
+      {
+        encoding: "hex",
+        parsed: true,
+        ignoreInvalidPriceIds: true,
+      }
+    );
+
+    eventSource.onerror = (event: Event) => {
+      console.error("Hermes streaming error", event);
+    };
+
+    /// Await for the first message before continuing
+    await new Promise<void>((resolve, reject) => {
+      setTimeout(() => {
+        reject(new Error("Hermes streaming timeout"));
+      }, argv.hermesStreamingTimeout);
+
+      eventSource.onmessage = (event: MessageEvent<string>) => {
+        resolve();
+
+        const data: PriceUpdate = JSON.parse(event.data);
+        const now = Date.now();
+        if (data.parsed) {
+          for (const parsedUpdate of data.parsed) {
+            const priceConfig = priceConfigs.find(
+              (priceConfig) => priceConfig.pythFeedId === parsedUpdate.id
+            );
+            if (priceConfig) {
+              if (
```

Comment:
i think you should store the price no matter how stale it is, as long as it's fresher than what is currently stored. i think you can handle processing whether to use the prices in the price evaluation

Reply:
done